rewrite-java/src/main/java/org/openrewrite/java/nullability/StandardizeNullabilityAnnotations.java

```diff
@@ -0,0 +1,183 @@
+            nullable("com.sun.istack.Nullable"),
+            nullable("com.sun.istack.internal.Nullable"),
+            nullable("com.unboundid.util.Nullable"),
+            nullable("edu.umd.cs.findbugs.annotations.CheckForNull"),
+            nullable("edu.umd.cs.findbugs.annotations.Nullable"),
+            nullable("edu.umd.cs.findbugs.annotations.PossiblyNull"),
+            nullable("edu.umd.cs.findbugs.annotations.UnknownNullness"),
+            nullable("io.micrometer.core.lang.Nullable"),
+            nullable("io.reactivex.annotations.Nullable"),
+            nullable("io.reactivex.rxjava3.annotations.Nullable"),
+            nullable("javax.annotation.CheckForNull"),
+            nullable("javax.annotation.Nullable").withAllTargets().withAllScopes(),
+            nullable("junitparams.converters.Nullable"),
+            nullable("libcore.util.Nullable"),
+            nullable("org.apache.avro.reflect.Nullable"),
+            nullable("org.apache.cxf.jaxrs.ext.Nullable"),
+            nullable("org.apache.shindig.common.Nullable"),
+            nullable("org.checkerframework.checker.nullness.compatqual.NullableDecl"),
+            nullable("org.checkerframework.checker.nullness.compatqual.NullableType"),
+            nullable("org.checkerframework.checker.nullness.qual.Nullable"),
+            nullable("org.codehaus.commons.nullanalysis.Nullable"),
+            nullable("org.eclipse.jdt.annotation.Nullable"),
+            nullable("org.eclipse.jgit.annotations.Nullable"),
+            nullable("org.jetbrains.annotations.Nullable"),
+            nullable("org.jetbrains.annotations.UnknownNullability"),
+            nullable("org.jmlspecs.annotation.Nullable"),
+            nullable("org.jspecify.nullness.Nullable"),
+            nullable("org.jspecify.nullness.NullnessUnspecified"),
+            nullable("org.netbeans.api.annotations.common.CheckForNull"),
+            nullable("org.netbeans.api.annotations.common.NullAllowed"),
+            nullable("org.netbeans.api.annotations.common.NullUnknown"),
+            nullable("org.openrewrite.internal.lang.Nullable").withAllTargets().withAllScopes(),
+            nullable("org.openrewrite.internal.lang.NullFields").withTargets(ElementType.PACKAGE, ElementType.TYPE).withScopes(NullabilityAnnotation.Scope.FIELD),
+            nullable("org.springframework.lang.Nullable").withAllTargets().withAllScopes(),
+            nullable("reactor.util.annotation.Nullable")
+    ).collect(Collectors.toSet());
+
+    @Option(displayName = "Nullability annotations to use",
+            description = "All other nullability annotations will be replaced by these.",
+            example = "javax.annotation.Nullable")
+    List<String> nullabilityAnnotationsFqn;
+
+    @Option(displayName = "Nullability annotations to use",
+            description = "All other nullability annotations will be replaced by these.",
+            required = false)
+    Set<NullabilityAnnotation> additionalNullabilityAnnotations;
```

Comment:
I think it would make sense to change the `displayName` and `description` slightly, so that it is clear what the difference is between `nullabilityAnnotationsFqn` and `additionalNullabilityAnnotations`.

Reply:
Oh, forgot to change these, thank you spotting!